spec/pascal_spec.rb

```diff
@@ -2,28 +2,28 @@
 
 describe Pascal do
   describe ".nth_row" do
-    context "run without argument" do
+    context "without argument" do
       it "returns first row of pascal triangle" do
         sample = Pascal.new
         expect(sample.nth_row).to eql("1")
       end
     end
 
-    context "run argument: 1" do
+    context "argument: 1" do
```

Comment:
Context usually starts with some of the words like with, without, when, ... It would be better if this context description and similar ones look something like this `context "with argument 1"`.